core/src/main/scala/kafka/zk/ZkData.scala

```diff
@@ -460,30 +462,30 @@ case class ZkAclStore(nameType: ResourceNameType) {
   val aclPath: String = nameType match {
     case Literal => "/kafka-acl"
     case Prefixed => "/kafka-prefixed-acl"
-    case _ => throw new IllegalArgumentException("Unknown name type:" + nameType)
-  }
-
-  val aclChangePath: String = nameType match {
-    case Literal => "/kafka-acl-changes"
-    case Prefixed => "/kafka-prefixed-acl-changes"
-    case _ => throw new IllegalArgumentException("Unknown name type:" + nameType)
+    case _ => throw new IllegalArgumentException("Invalid name type:" + nameType)
   }
 
   def path(resourceType: ResourceType) = s"$aclPath/$resourceType"
 
   def path(resourceType: ResourceType, resourceName: String): String = s"$aclPath/$resourceType/$resourceName"
-
-  def changeSequenceZNode: AclChangeNotificationSequenceZNode = AclChangeNotificationSequenceZNode(this)
-
-  def decode(notificationMessage: Array[Byte]): Resource = AclChangeNotificationSequenceZNode.decode(nameType, notificationMessage)
 }
 
 object ZkAclStore {
   val stores: Seq[ZkAclStore] = ResourceNameType.values
     .map(nameType => ZkAclStore(nameType))
 
   val securePaths: Seq[String] = stores
-    .flatMap(store => List(store.aclPath, store.aclChangePath))
+    .flatMap(store => List(store.aclPath)) ++ Seq(AclChangeNotificationZNode.path)
+}
+
+@deprecated("There are now multiple roots for ACLs within ZK. Use ZkAclStore", "2.0")
+object AclZNode {
+  def path = "/kafka-acl"
+}
+
+@deprecated("There are now multiple roots for ACLs within ZK. Use ZkAclStore", "2.0")
+object ResourceTypeZNode {
```

Comment:
AclZNode and ResourceTypeZNode are not part of the public API. We can just remove them.

Reply:
Done.